File: src/java/ifelse.java
public class ifelse {

	public static void main(String[] args) {

		boolean value = true;
		if (value) {
			System.out.println("Got true");
		} else {
			System.out.println("Got else");
		}

		value = false;
		if (value)
			System.out.println("Got true");
		else
			System.out.println("Got else");

		int result = 3;
		if (result == 1)
			System.out.println("Got true");
		else if (result == 2)
			System.out.println("Got else");
		else
			System.out.println("Got if else");
	}

}

File: src/scala/ifelse.scala
object IfElse extends App {
    
    var value = true
    if (value) {
        println("Got true")
    } else {
        println("Got else")
    }

    value = false
    if (value)
        println("Got true")
    else
        println("Got else")

    val result = 3
    if (result == 1)
        println("Got true")
    else if (result == 2)
        println("Got else")
    else
        println("Got if else")
}